low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -340,3 +342,19 @@ def get_nw_status(self, nw_interface, interface):
             nw_cable_conn_status = "UNKNOWN"
             # Log the error when logging class is in place.
         return nw_status, nw_cable_conn_status
+
+    def get_raid_info(self):
+        raids_data = []
+        for raid in RAIDs.get_configured_raids():
+            raid_data = self.get_health_template(raid.id, False)
+            health, description = raid.get_health()
+            devices = []
+            if health != "Fault":
+                devices = raid.get_devices()
+            specifics = [{"location": raid.raid,
+                          "data_integrity_status": raid.get_data_integrity_status(),
+                          "devices": devices}]
+            self.set_health_data(raid_data, health, specifics=specifics, description=description)
+            raid_data["last_updated"] = int(time.time())
```

Comment:
This is not needed, set_health_data function takes care of adding 'last_updated' filed in the schema,

Reply:
Oh good. In worst this will overwrite that time ...

Reply:
Done